lib/eu_central_bank.rb

```diff
@@ -136,6 +139,12 @@ def import_rates(format, s, opts = {})
     self
   end
 
+  def check_currency_available(currency)
+    return true if currency == "EUR"
+    return true if CURRENCIES.include?(currency)
+    raise CurrencyUnavailable
```

Comment:
You definitely wanna give it an error message with currency that is unavailable, like `"No rates available for #{currency}"`, which makes it much more clear what went wrong